feature-libs/customer-ticketing/root/model/customer-ticketing.model.ts

```diff
@@ -29,6 +29,20 @@ export interface Category {
   name: string;
 }
 
+export interface CategoryList {
```

Comment:
```suggestion
export interface CategoriesList {
```
Or ListOfCategories
Or categoryListing